### PR TITLE
SCSS variables --> CSS variables

### PR DESCRIPTION
**Pull request by ludwiczakpawel**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [x] Optimization

## Description

This PR translated our SCSS variables into CSS native variables. It should not affect visually any part of the site.

**Why?**
CSS variables are much more flexible when it comes to responsiveness and building components. Browsers' support for this technology is wide enough ([94%](https://caniuse.com/#feat=css-variables)) to call it a _safe choice_.

## Added tests?

- [x] no, because they aren't needed

## Added to documentation?

- [x] no documentation needed

### CODE REVIEW

File: app/assets/stylesheets/config/_import.scss [hunk 1]
```diff
@@ -58,9 +47,7 @@ $site-width: 1280px;
 $breakpoint-s: 640px;
 $breakpoint-m: 768px;
 $breakpoint-l: 1024px;
-$breakpoint-xl: $site-width;
-
-////////////////////////////////////////////////////////////////////////////////////
+$breakpoint-xl: 1280px;
```
Comment (ludwiczakpawel) [inline app/assets/stylesheets/config/_import.scss:50]: Fun fact: you can't use CSS variables with `@media queries`

File: app/assets/stylesheets/base/layout.scss [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
   --layout-sidebar-width: 200px;
-  --layout-gap: #{$su-6};
-  --layout-padding: #{$su-6};
+  --layout-gap: var(--su-6);
+  --layout-padding: var(--su-6);
 }
 
 .crayons-layout {
-  font-size: $fs-base; // todo: drop it when ready
+  font-size: var(--fs-base); // todo: drop it when ready
```
Comment (rhymes) [inline app/assets/stylesheets/base/layout.scss:10]: are we ready? 🤣

Comment (ludwiczakpawel) [reply app/assets/stylesheets/base/layout.scss:10]: YOLO!

**Comment by lisasy** [comment]
Fascinating!

Will you get the chance to share/educate everyone in the company on this new best practices? I think it's great and makes a lot of sense to use native CSS variables.

**Comment by benhalpern** [comment]
Ack. We'll need to fix the conflicts.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -119,14 +119,15 @@ score.
```
Comment (rhymes) [inline CONTRIBUTING.md:119]: I don't think this belongs in this PR

Comment (ludwiczakpawel) [reply CONTRIBUTING.md:119]: uh... no idea...

**Comment by ludwiczakpawel** [comment]
@rhymes correct. fix is already in.

**Pull request merged**